docs/samples/custom/timeseries-holtwinter/custom.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: serving.kubeflow.org/v1alpha2
+kind: InferenceService
+metadata:
+  labels:
+    controller-tools.k8s.io: "1.0"
+  name: sales-application
+spec:
+  default:
+    predictor:
+      custom:
+        name: custom
+        container:
+          image: {docker-username}/kfserving-custom-example
```

Comment:
now you can add the storage uri env to download the model automatically to /mnt/model/model_holt.sav
```
apiVersion: serving.kubeflow.org/v1alpha2
kind: InferenceService
metadata:
  labels:
    controller-tools.k8s.io: "1.0"
  name: sales-application
spec:
  default:
    predictor:
      custom:
        name: custom
        container:
          image: {docker-username}/kfserving-custom-example
          env:
          - name: STORAGE_URI
             value: gs://kfserving-examples/models/statsmodel
```